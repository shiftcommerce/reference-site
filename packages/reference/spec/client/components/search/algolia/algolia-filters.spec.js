// Components
import AlgoliaFilters from '../../../../../client/components/search/algolia/algolia-filters'

test('renders the algolia filter component with children', () => {
  // Arrange & Act
  const wrapper = shallow(<AlgoliaFilters showFilters={true} />)

  // Assert
  expect(wrapper).toMatchSnapshot()
})
