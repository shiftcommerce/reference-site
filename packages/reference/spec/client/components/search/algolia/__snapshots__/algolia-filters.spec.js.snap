// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders the algolia filter component with children 1`] = `
<div
  className="c-product-listing-filter"
>
  <div
    className="c-product-listing-filter__header"
  >
    <div
      className="c-product-listing-filter__header-title"
    >
      <h2>
        Filters
      </h2>
      <button
        className="c-product-listing-filter-close"
      />
    </div>
    <div
      className="c-product-listing-filter__header-selection"
    >
      <AlgoliaCurrentRefinements(ClearRefinementsWidget) />
      <AlgoliaCurrentRefinements(CurrentRefinementsWidget)
        transformItems={[Function]}
      />
    </div>
  </div>
  <Panel
    className="c-product-listing-filter__body-option"
    footer={null}
    header={
      <h2>
        Sub Category
      </h2>
    }
  >
    <AlgoliaRefinementList(RefinementListWidget)
      attribute="category_ids"
      limit={3}
      operator="or"
      showMore={true}
      showMoreLimit={20}
      transformItems={[Function]}
    />
  </Panel>
  <Panel
    className="c-product-listing-filter__body-option"
    footer={null}
    header={
      <h2>
        Rating
      </h2>
    }
  >
    <AlgoliaRefinementList(AlgoliaRatingFilter)
      attributeName="product_rating"
      limitMax={20}
      limitMin={10}
      max={5}
      min={0}
      operator="or"
      showMore={false}
    />
  </Panel>
  <Panel
    className="c-product-listing-filter__body-option"
    footer={null}
    header={
      <h2>
        Price
      </h2>
    }
  >
    <AlgoliaRange(RangeSlider)
      attribute="variant_meta_data.eu.price"
      formatLabel={[Function]}
      precision={0}
    />
  </Panel>
</div>
`;
